classes/main.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3
+
+import time
+
+'''
+
+In this game we have two role - invader and hero. Both can bullet.
+
+Invaders are located at the top of game field and always move from the right side to the left and revert.
+Also they slowly move to the bottom.
+
+Main hero is located at the centre at the bottom. He can move to the left and to the right.
+If he bullet some invader, his bonus is grow.
+When invader bullet to main hero, the main hero's life is decrease.
+
+'''
+
+
+class GameScreen(object):
+
+    def __init__(self, filename, height, width):
+        self.image = filename
+        self.height = height
+        self.width = width
+
+
+class Unit(object):
+
+    def __init__(self, filename, x, y, bonus=0, speed=5):
+        self.image = filename
+        self.x = x
+        self.y = y
+        self.width = 10  # temporary when we don't have real images
+        self.height = 10  # must be height of real image
+        self.bonus = bonus
+        self.speed = speed
+        self.is_dead = False
+        self.bullet = Bullet(self)
+        self.shift = 10
+
+    def check_explosion(self, other_unit):
+        pass
+
+    def make_explosion(self, other_unit):
+        pass
+
+    def make_bullet(self):
+        pass
+
+    def set_bullet_image(self):
+        pass
+
+
+class Invader(Unit):
+    moving_speed = 0
+
+    def move_x(self, speed):
```

Comment:
every unit must be able to move. We need a method like "move_to(x, y)" that accepts new coordinates to move
This action must keep information like time of new coordinates were accepted and current coordinate on the moment of call. Based to that information (unit's speed, previous coordinates and new coordinates) we can extract current position of  every unit